spec/spec_helper.rb

```diff
@@ -3,6 +3,7 @@
 require 'json'
 require 'yaml'
 require 'English'
+require 'byebug'
```

Comment:
Was this addition intentional?

Reply:
Yes I like it being available so I can add `byebug` anywhere to get a debugger. Is it a bad practice?

Reply:
I have a key bound to insert `require 'pry'; binding.pry`. :)

A linter would probably complain about an unused `require`, but given that it's in `spec/spec_helper.rb`, it doesn't seem like it matters very much.